quint/src/cliCommands.ts

```diff
@@ -66,7 +66,7 @@ interface OutputStage {
   failed?: string[]
   ignored?: string[]
   // Test names output produced by 'run'
```

Comment:
I know it wasn't introduced in your changes, but it looks like this comment corresponds to the field you are updating. Do you suppose this is out of place or out of date? Or do you know what it means w/r/t `status`?

Reply:
Seems like it is out of date, I'll try to find out more in the history

Reply:
Cool cool. Super low priority. I'm fine with us getting this fix in and cutting a release ASAP, if you are. I agree that this is a very bad and we should patch it up.

Reply:
Seems like it came from here https://github.com/informalsystems/quint/commit/6ebf042f5117e6a28013122f7daf0b359d6447df#diff-229e3647dd4a8f61338fb58a6266e6fca5690a1dcb1a0550cd0c1fba2b372d7dR53

I'd rule this as a copy-paste mistake (copy pasted from the "// Test names output produced by 'test'" and then it was never updated to match the actual meaning of this attribute)